CliClient/locales/de_DE.po

```diff
@@ -1357,27 +1361,32 @@ msgstr "ID"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:168
 msgid "Upgrade"
-msgstr ""
+msgstr "Upgrade"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:181
-#, fuzzy
 msgid "Re-encrypt data"
-msgstr "Datum auswählen"
+msgstr "Daten neu verschlüsseln"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:183
+#, fuzzy
 msgid ""
 "The default encryption method has been changed to a more secure one and it "
```

Comment:
Why did you add `fuzzy` there? In Poedit this is the flag `Needs Works`.

This means that the translation won't be used.